etna/media/blocks.py

```diff
@@ -3,34 +3,23 @@
 from django.template.loader import render_to_string
 
 from wagtail.core import blocks
-from wagtailmedia.blocks import ChooserBlock, AbstractMediaChooserBlock
+from wagtail.images.blocks import ImageChooserBlock
+from wagtailmedia.blocks import AbstractMediaChooserBlock
 
 
 class EtnaMediaBlock(AbstractMediaChooserBlock):
     def render_basic(self, value, context=None):
-        if not value:
-            return ""
+        pass
```

Comment:
Would you mind adding a comment about why this method is required even though it does nothing.

I checked the wagtail source, render_basic is only called if no template is defined:

https://github.com/wagtail/wagtail/blob/8413d00bdd03c447900019961d604186e17d2870/wagtail/core/blocks/base.py#L206

Reply:
Done